src/styles.css/reset.css

```
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
don't write anything else except `box-sizing: border-box;` to `*`

---

if i delete margin, my picture is brokes. if I move it to body, the picture breaks too

---

reset margins for each element which you need to but not using `*` selector